tsl/test/expected/continuous_aggs_query-11.out

```diff
@@ -721,3 +721,143 @@ select * from mat_m1 order by timec desc, location;
+         to_timestamp         
+------------------------------
+ Sat Nov 03 17:00:00 2018 PDT
+(1 row)
+
+-- Drop the most recent chunk
+SELECT chunk_name, range_start, range_end
+FROM timescaledb_information.chunks
+WHERE hypertable_name = :'mat_name';
+    chunk_name    |         range_start          |          range_end           
+------------------+------------------------------+------------------------------
+ _hyper_2_3_chunk | Wed Nov 29 16:00:00 2017 PST | Wed Feb 07 16:00:00 2018 PST
+ _hyper_2_4_chunk | Wed Sep 05 17:00:00 2018 PDT | Wed Nov 14 16:00:00 2018 PST
+(2 rows)
+
+SELECT drop_chunks('mat_m1', newer_than=>'2018-01-01'::timestamptz);
+              drop_chunks               
+----------------------------------------
+ _timescaledb_internal._hyper_2_4_chunk
+(1 row)
+
+SELECT chunk_name, range_start, range_end
+FROM timescaledb_information.chunks
+WHERE hypertable_name = :'mat_name';
+    chunk_name    |         range_start          |          range_end           
+------------------+------------------------------+------------------------------
+ _hyper_2_3_chunk | Wed Nov 29 16:00:00 2017 PST | Wed Feb 07 16:00:00 2018 PST
+(1 row)
+
+-- The watermark should be updated to reflect the dropped data (i.e.,
+-- the cache should be reset)
+SELECT _timescaledb_internal.to_timestamp(_timescaledb_internal.cagg_watermark(:mat_id));
+         to_timestamp         
+------------------------------
+ Tue Jan 02 16:00:00 2018 PST
+(1 row)
+
+-- Try a subtransaction
+SAVEPOINT clear_cagg;
+SELECT m1.location, m1.timec, sumt, sumh, firsth, lasth, maxtemp, mintemp
+FROM mat_m1 m1, mat_m2 m2
+WHERE m1.location = m2.location
+AND m1.timec = m2.timec
+ORDER BY m1.location COLLATE "C", m1.timec DESC
+LIMIT 10;
+ location |            timec             | sumt | sumh | firsth | lasth | maxtemp | mintemp 
+----------+------------------------------+------+------+--------+-------+---------+---------
+ NYC      | Sun Dec 02 16:00:00 2018 PST |  110 |   75 |     45 |    30 |      65 |      45
+ NYC      | Fri Nov 02 17:00:00 2018 PDT |      |      |        |       |         |        
+ NYC      | Thu Nov 01 17:00:00 2018 PDT |   30 |   25 |     10 |       |      20 |      10
+ NYC      | Wed Oct 31 17:00:00 2018 PDT |  325 |  200 |     30 |    50 |      85 |      45
+ NYC      | Mon Jan 01 16:00:00 2018 PST |   65 |   45 |     45 |    45 |      65 |      65
+ SFO      | Sun Dec 02 16:00:00 2018 PST |  120 |   90 |     45 |    45 |      65 |      55
+ SFO      | Mon Jan 01 16:00:00 2018 PST |   65 |   45 |     45 |    45 |      65 |      65
+ SFO      | Sun Dec 31 16:00:00 2017 PST |   55 |   45 |     45 |    45 |      55 |      55
+ por      | Mon Jan 01 16:00:00 2018 PST |  100 |  100 |    100 |   100 |     100 |     100
+(9 rows)
+
+ALTER MATERIALIZED VIEW mat_m1 SET (timescaledb.materialized_only=true);
+SELECT m1.location, m1.timec, sumt, sumh, firsth, lasth, maxtemp, mintemp
```

Comment:
SELECT m2.location, m2.timec, sumt, sumh, firsth, lasth, maxtem, mintemp
FROM mat_m1 RIGHT JOIN mat_m2 ON ....

This would also verify/highlight the difference between the 2  caggs.

Reply:
Done.